admin/broadleaf-open-admin-platform/src/main/java/org/broadleafcommerce/openadmin/web/controller/AdminAbstractController.java

```diff
@@ -464,10 +464,10 @@ protected FilterAndSortCriteria[] getCriteria(Map<String, List<String>> requestP
                     List<String> values = entry.getValue();
                     List<String> collapsedValues = new ArrayList<>();
                     for (String value : values) {
-                        String decoded = null;
+                        String decoded = value;
                         try {
                             decoded = eps.cleanString(URLDecoder.decode(value, "UTF-8"));
-                        } catch (UnsupportedEncodingException | ServiceException e) {
+                        } catch (Exception e) {
```

Comment:
I think we should try to encode % after decoding...so sql will treat % as a regular symbol not special char that means _any_ zero or more